packages/start/server/rpc.tsx

```diff
@@ -0,0 +1,8 @@
+import { getRequestEvent } from "solid-js/web";
+import type { RpcInfo } from "./types";
+
+export function getRpcInfo(): RpcInfo | undefined {
+  const event = getRequestEvent();
+
+  return event?.context.rpcInfo;
+}
```

Comment:
maybe this could just be done in user land? not sure

Reply:
Yes technically speaking it absolutely could 🙂. The reason why I added this little helper is because the event context type does not have an rpcInfo and I am not sure if/how we can extend the type. Having a helper seems like the easiest way to document the existence of rpcInfo in solid-start 🤔

Reply:
(https://github.com/ryansolid/dom-expressions/blob/0aeca28611b5e9d85b135efbfc39892672efac95/packages/dom-expressions/src/server.js#L570)